docs/how-to-guides/auto-update-flatcar.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Auto-updating Flatcar Container Linux
+weight: 10
+---
+
+## Introduction
+
+Lokomotive supports Flatcar Container Linux only. Flatcar Linux Update Operator (FLUO) can keep the operating system updated every time there is a new Flatcar release. FLUO is a supported component of the Lokomotive's component stack. This document will show how to install it and disable its auto-update feature for specific nodes.
+
+## Prerequisites
+
+- A Lokomotive cluster accessible via `kubectl`.
+
+## Steps
+
+### Step 1: Disable auto-update on nodes
+
+If you want to update the Flatcar OS of particular nodes in a controlled fashion, there is a way to disable such updates. Disabling updates can come handy when the workloads run by these machines are storage related or ingress nodes, where you cannot afford the slightest loss of connection.
+
+Please annotate the nodes with the following annotation:
+
+```bash
+kubectl annotate node <node name> "flatcar-linux-update.v1.flatcar-linux.net/reboot-paused=true"
+```
+
+### Step 2: Configure FLUO
+
+Create a file named `fluo.lokocfg` with the following contents:
```

Comment:
I'd make a file name an example, as it's not a hard requirement. Say: "Add FLUO component to your cluster configuration, for example by creating `fluo.lokocfg` file with the following content".